### PR TITLE
Jeff-approved intro text for numenta.org

### PR DESCRIPTION
**Pull request by rhyolight**

### CODE REVIEW

File: packages/numenta.org/pages/_Section.jsx [hunk 1]
```diff
@@ -108,37 +108,38 @@ const SectionHome = (props, {config}) => {
...
-          that provides a framework for both biological and machine
-          intelligence. Parts of HTM theory have been implemented, tested, and
-          used in applications, and other parts are still being developed.
+            Numenta.com
+          </TextLink> {' '}.
+        </Paragraph>
+
+        <Paragraph>
+          <Strong>Machine Intelligence Starts Here</Strong>
         </Paragraph>
```
Comment (rhyolight) [inline packages/numenta.org/pages/_Section.jsx]: Ok @brev same questions. 

1. How can I make this `<p>` into an `<h2>` using this markup?
2. What is this markup and where can I find docs?

Thanks!

File: packages/numenta.org/pages/_Section.jsx [hunk 2]
```diff
@@ -108,37 +108,38 @@ const SectionHome = (props, {config}) => {
       <div className={styles.content}>
 
         <Paragraph lead={true}>
-          Welcome to <Strong>Numenta.org,</Strong> an open platform and
-          community dedicated to advancing the state of brain theory and machine
-          intelligence.  On this site, you will find the technical resources you
-          need to get started with Hierarchical Temporal Memory (HTM). HTM is
-          a theory of the neocortex developed by {' '}
+          Welcome to <Strong>Numenta.org</Strong>, home of Numenta’s HTM community and open source project. If you want to learn about Numenta the company visit {' '}
```
Comment (brev) [inline packages/numenta.org/pages/_Section.jsx]: wrap at 80-wide please

**Comment by brev** [comment]
👍 looks good so far.  Sorry about the Section titles being in 3 places, I plan on refactoring those ASAP :)

**Comment by rhyolight** [comment]
I have some lint issues to clean up... standby

**Comment by rhyolight** [comment]
@brev do you know how I can add `SubTitle` to allowed React markup?

```
  134:10  error  'SubTitle' is not defined    react/jsx-no-undef
```

screen shot 2017-01-05 at 4 02 45 pm

**Comment by brev** [comment]
@rhyolight at the top:
```
import SubTitle from 'numenta-web-shared-components/lib/SubTitle'  
```

**Comment by rhyolight** [comment]
Ready @brev

**Comment by brev** [comment]
@rhyolight please run `npm run test:unit` from root to see new unit test failures.

**Comment by rhyolight** [comment]
got sum tests ta fix

**Comment by rhyolight** [comment]
@brev ok ready again (tests passing)

**Comment by brev** [comment]
👍 thanks

**Pull request merged**